AccountService.js

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import axios from 'axios';
+const headers = {Accept: 'application/ld+json, application/json'};
+export class AccountService {
+  constructor({
+    urls = {
+      base: '/accounts'
+    }
+  } = {}) {
+    this.config = {urls};
+  }
+
+  /**
+  * @method exists
+  * @param {Object} config
+  * @param {String} config.baseUrl
+  * @param {String} config.email
+  * @return {Boolean} exists
+  * @description the api retruns 404 if the user does not exist
+  * if the request is succesfull returns true
+  */
+  async exists({baseUrl = this.config.urls.base, email}) {
+    try {
+      await axios.get(baseUrl, {
+        params: {
+          exists: true,
+          email
+        }
+      });
+      return true;
+    } catch(e) {
+      if(!e.response) {
```

Comment:
how about this
```
if(e.response && e.response.status && e.response.status === 404) {
  return false;
}
throw e;
```

Reply:
more concise I like it.

Reply:
Seems like this would suffice:

```js
if(e.response && e.response.status === 404) {
  return false;
}
throw e;
```

Reply:
Now, I think it would be even better to inspect the contents of `e.response.data` and ensure that it's a NotFoundError as expected there.  A 404 could be for other reasons.  The only way to be sure it's the 404 we were expecting is to look for the error in the data.

Reply:
404 could be because the route was not found I believe.
So yeah I guess we should look at the data.